src/sentry/static/sentry/app/views/alerts/details/body.tsx

```diff
@@ -207,23 +207,23 @@ export default class DetailsBody extends React.Component<Props> {
             <PageContent>
               {incident?.alertRule && (
                 <React.Fragment>
-                  <SideHeader>
+                  <SidebarHeading>
                     <span>{t('Alert Rule')}</span>
                     {incident?.alertRule?.status !== AlertRuleStatus.SNAPSHOT && (
                       <SideHeaderLink
                         to={{
                           pathname: `/settings/${params.orgId}/projects/${incident?.projects[0]}/alerts/metric-rules/${incident?.alertRule?.id}/`,
                         }}
                       >
-                        <IconEdit />
                         {t('View Rule')}
+                        <IconLink size="xs" />
```

Comment:
The link looks a bit more visually matching appealing I think and makes more sense in context of 'view rule'. If this was 'Edit rule' I could be more convinced.

Moved to the right for visual consistency with the discover icon.